src/os/bluestore/BlueStore.cc

```diff
@@ -6293,7 +6293,22 @@ string BlueStore::get_device_path(unsigned id)
   return res;
 }
 
-int BlueStore::expand_devices(ostream& out)
+int BlueStore::check_during_expand() {
+  bluestore_bdev_label_t label;
+  string p = path + "/block";
+  int r = _read_bdev_label(cct, p, &label);
+  if (r < 0) {
+    derr << __func__ << " unable to read label from device " << p << dendl;
+    return -EIO;
+  }
+  if (label.during_emergency_expand) {
+    derr << __func__ << " device " << p << " has unfinished emergency expand." << dendl;
```

Comment:
Suggest to append 'how to fix this' description (e.g. 'please rerun: ceph-bluestore-tool bluefs-bdev-expand ...' to the error message.